### PR TITLE
[release] release 1.3.0 results and test updates

### PR DESCRIPTION
**Pull request by krfricke**
## Related issue number

Replaces #14995

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by krfricke** [comment]
This should be ready for review (failing tests seem unrelated)

File: .bazelrc [hunk 1]
```diff
@@ -94,7 +94,7 @@ build:asan --linkopt -fsanitize=address
 test:asan --jobs=1
 test:asan --test_env=ASAN_OPTIONS="detect_leaks=0"
 # This LD_PRELOAD is set for Travis. You will need to change it for local debugging.
-test:asan --test_env=LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libasan.so.2 /usr/lib/gcc/x86_64-linux-gnu/7/libasan.so"
+test:asan --test_env=LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libasan.so.5 /usr/lib/gcc/x86_64-linux-gnu/9/libasan.so"
```
Comment (richardliaw) [inline .bazelrc:97]: who is the relevant person to review this? cc @ericl ?

File: release/release_logs/1.3.0/rllib_tests/unit_gpu_tests.txt [hunk 2]
```diff
@@ -0,0 +1,2135 @@
+Script started on 2021-04-09 04:56:02-0700
```
Comment (richardliaw) [inline release/release_logs/1.3.0/rllib_tests/unit_gpu_tests.txt:1]: can we remove this? I don't think we've saved this in the past.

File: release/release_logs/1.3.0/xgboost_tests/ft_small_elastic.txt [hunk 3]
```diff
@@ -0,0 +1,220 @@
+Script started on 2021-03-30 04:50:46-07:00 [TERM="screen" TTY="/dev/pts/11" COLUMNS="80" LINES="24"]
```
Comment (richardliaw) [inline release/release_logs/1.3.0/xgboost_tests/ft_small_elastic.txt:1]: hmm does it make sense to save this too?

File: release/microbenchmark/commits.txt [hunk 4]
```diff
@@ -0,0 +1,606 @@
+8ed3de051147b559f296b8f697eb6839145f6e38
```
Comment (richardliaw) [inline release/microbenchmark/commits.txt:1]: do we need this?

**Review by richardliaw — CHANGES_REQUESTED** [review]
I think historically we haven't kept the logs of every test (and we probably don't need to). Could you remove some of the longer test logs (i.e., the xgboost release test logs)?

**Review by wuisawesome — COMMENTED** [review]
I agree with Richard. Essentially the diff for this should be like 100 LoC (maybe a bit longer since there are some nice improvements here) but not 10k.

File: release/release_logs/1.3.0/xgboost_tests/ft_small_elastic.txt (hunk 3 above)
Comment (krfricke) [reply release/release_logs/1.3.0/xgboost_tests/ft_small_elastic.txt]: Happy to remove these if you think this is unnecessary information - we're not collecting performance measures here, so I guess it's not that important to keep them

File: .bazelrc (hunk 1 above)
Comment (krfricke) [reply .bazelrc:97]: FWIW these are only the asan tests and they only seem to be run manually. The same change has been done for buildkite:

```
test:asan-buildkite --test_env=LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libasan.so.5"
```

**Comment by krfricke** [comment]
I removed a number of logs. Unfortunately the bulk of the changes will have to remain in place because we had to change the respective release tests. Would it help to split this into two PRs (one for the changes, one for the logs)?

File: rllib/BUILD [hunk 5]
```diff
@@ -1729,7 +1729,7 @@ py_test(
     name = "examples/batch_norm_model_ddpg_tf",
     main = "examples/batch_norm_model.py",
     tags = ["examples", "examples_B"],
-    size = "small",
+    size = "medium",
```
Comment (richardliaw) [inline rllib/BUILD:1732]: should we revert this?

**Review by richardliaw — APPROVED** [review]
Looks good, just one nit

File: rllib/BUILD (hunk 5 above)
Comment (krfricke) [reply rllib/BUILD:1732]: No, it doesn't pass without this. Torch is already at medium and IIRC this was flaky

**Pull request merged**